2025-04-14 16:23:29,606 - cephci - log:161 - DEBUG - Completed log configuration
2025-04-14 16:23:29,609 - cephci - run:722 - INFO - Running test cephfs_nfs.nfs_ro_rw_access.py
2025-04-14 16:23:29,610 - cephci - nfs_ro_rw_access:44 - INFO - Running cephfs CEPH-83574001 test case
2025-04-14 16:23:29,611 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.251
2025-04-14 16:23:30,615 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 16:23:31,207 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:23:31,208 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 16:23:31,208 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:23:32,079 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 2:47:38 ago on Mon Apr 14 13:35:33 2025.
2025-04-14 16:23:32,341 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:23:32,342 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 16:23:32,344 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 16:23:32,345 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 16:23:32,346 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 16:23:32,347 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 16:23:32,348 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 16:23:32,349 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:23:32,350 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 16:23:32,351 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 16:23:32,446 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 16:23:32,447 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 16:23:32,447 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 16:23:32,584 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.251 took 2.972743 seconds
2025-04-14 16:23:32,627 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.251
2025-04-14 16:23:33,631 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 16:23:33,631 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 16:23:33,631 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 16:23:33,631 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 16:23:33,632 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 16:23:33,632 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 16:23:33,632 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.251 took 1.005142 seconds
2025-04-14 16:23:33,634 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.251
2025-04-14 16:23:34,639 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.251 took 1.004043 seconds
2025-04-14 16:23:34,641 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.251
2025-04-14 16:23:35,645 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.251 took 1.003712 seconds
2025-04-14 16:23:35,647 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.3
2025-04-14 16:23:36,650 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 16:23:37,170 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:23:37,170 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 16:23:37,170 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:23:38,042 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 2:27:11 ago on Mon Apr 14 13:56:06 2025.
2025-04-14 16:23:38,275 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:23:38,276 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 16:23:38,277 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 16:23:38,278 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 16:23:38,279 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 16:23:38,280 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 16:23:38,281 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 16:23:38,283 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:23:38,284 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 16:23:38,285 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 16:23:38,377 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 16:23:38,379 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 16:23:38,379 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 16:23:38,494 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.3 took 2.846533 seconds
2025-04-14 16:23:38,536 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.3
2025-04-14 16:23:39,539 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 16:23:39,540 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 16:23:39,540 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 16:23:39,540 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 16:23:39,540 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 16:23:39,540 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 16:23:39,540 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.3 took 1.004268 seconds
2025-04-14 16:23:39,542 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.3
2025-04-14 16:23:40,545 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.3 took 1.003123 seconds
2025-04-14 16:23:40,547 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.3
2025-04-14 16:23:41,551 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.3 took 1.003548 seconds
2025-04-14 16:23:41,553 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.202
2025-04-14 16:23:42,559 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 16:23:43,055 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:23:43,055 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 16:23:43,055 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:23:43,868 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 2:44:05 ago on Mon Apr 14 13:39:18 2025.
2025-04-14 16:23:44,097 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:23:44,098 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 16:23:44,099 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 16:23:44,101 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 16:23:44,102 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 16:23:44,103 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 16:23:44,104 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 16:23:44,105 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:23:44,106 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 16:23:44,107 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 16:23:44,199 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 16:23:44,200 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 16:23:44,200 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 16:23:44,337 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.202 took 2.783725 seconds
2025-04-14 16:23:44,379 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.202
2025-04-14 16:23:45,383 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 16:23:45,383 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 16:23:45,383 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 16:23:45,383 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 16:23:45,383 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 16:23:45,383 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 16:23:45,384 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.202 took 1.004607 seconds
2025-04-14 16:23:45,385 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.202
2025-04-14 16:23:46,390 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.202 took 1.004083 seconds
2025-04-14 16:23:46,392 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.202
2025-04-14 16:23:47,396 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.202 took 1.003592 seconds
2025-04-14 16:23:47,398 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.142
2025-04-14 16:23:48,405 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 16:23:48,885 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:23:48,885 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 16:23:48,885 - cephci - ceph:1186 - DEBUG - 
2025-04-14 16:23:49,646 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 2:41:23 ago on Mon Apr 14 13:42:05 2025.
2025-04-14 16:23:49,890 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:23:49,891 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 16:23:49,893 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 16:23:49,894 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 16:23:49,895 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 16:23:49,896 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 16:23:49,897 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 16:23:49,898 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 16:23:49,900 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 16:23:49,901 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 16:23:49,991 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 16:23:49,992 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 16:23:49,993 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 16:23:50,136 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.142 took 2.73716 seconds
2025-04-14 16:23:50,177 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.142
2025-04-14 16:23:51,181 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 16:23:51,182 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 16:23:51,182 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 16:23:51,182 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 16:23:51,182 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 16:23:51,182 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 16:23:51,182 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.142 took 1.004421 seconds
2025-04-14 16:23:51,184 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.142
2025-04-14 16:23:52,188 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.142 took 1.003689 seconds
2025-04-14 16:23:52,189 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.142
2025-04-14 16:23:53,193 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.142 took 1.003551 seconds
2025-04-14 16:23:53,195 - cephci - ceph:1576 - INFO - Execute ceph fs ls --format json-pretty on 10.0.195.251
2025-04-14 16:23:54,199 - cephci - ceph:1606 - INFO - Execution of ceph fs ls --format json-pretty on 10.0.195.251 took 1.003346 seconds
2025-04-14 16:23:54,200 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 16:23:54,201 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251
2025-04-14 16:23:55,204 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251 took 1.003046 seconds
2025-04-14 16:23:55,205 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251
2025-04-14 16:23:56,209 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251 took 1.002927 seconds
2025-04-14 16:23:56,210 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node8 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251
2025-04-14 16:23:57,214 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node8 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251 took 1.003039 seconds
2025-04-14 16:23:57,216 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251
2025-04-14 16:23:58,219 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251 took 1.003227 seconds
2025-04-14 16:23:58,220 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 16:23:58,222 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3
2025-04-14 16:23:59,225 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3 took 1.003482 seconds
2025-04-14 16:23:59,227 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3
2025-04-14 16:24:00,231 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3 took 1.003452 seconds
2025-04-14 16:24:00,232 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node9 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3
2025-04-14 16:24:01,236 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node9 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3 took 1.002996 seconds
2025-04-14 16:24:01,237 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3
2025-04-14 16:24:02,240 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3 took 1.002877 seconds
2025-04-14 16:24:02,241 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 16:24:02,242 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202
2025-04-14 16:24:03,246 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202 took 1.003559 seconds
2025-04-14 16:24:03,247 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202
2025-04-14 16:24:04,251 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202 took 1.003207 seconds
2025-04-14 16:24:04,253 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node10 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202
2025-04-14 16:24:06,258 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node10 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202 took 2.005026 seconds
2025-04-14 16:24:06,260 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202
2025-04-14 16:24:07,264 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202 took 1.00375 seconds
2025-04-14 16:24:07,265 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 16:24:07,266 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142
2025-04-14 16:24:08,270 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142 took 1.003636 seconds
2025-04-14 16:24:08,271 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142
2025-04-14 16:24:09,274 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142 took 1.002723 seconds
2025-04-14 16:24:09,276 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node11 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142
2025-04-14 16:24:10,280 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node11 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142 took 1.003167 seconds
2025-04-14 16:24:10,282 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142
2025-04-14 16:24:11,286 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142 took 1.003599 seconds
2025-04-14 16:24:11,289 - cephci - ceph:1576 - INFO - Execute ceph nfs cluster create cephfs-nfs ceph-regression-vhr823-futhjo-node7 on 10.0.195.251
2025-04-14 16:24:12,292 - cephci - ceph:1606 - INFO - Execution of ceph nfs cluster create cephfs-nfs ceph-regression-vhr823-futhjo-node7 on 10.0.195.251 took 1.003238 seconds
2025-04-14 16:24:12,293 - cephci - cephfs_volume_management:23 - INFO - Wait for the process to start or stop
2025-04-14 16:24:12,294 - cephci - ceph:1576 - INFO - Execute ceph orch ps | grep cephfs-nfs on 10.0.195.251
2025-04-14 16:24:13,298 - cephci - ceph:1606 - INFO - Execution of ceph orch ps | grep cephfs-nfs on 10.0.195.251 took 1.003432 seconds
2025-04-14 16:24:13,299 - cephci - ceph:1576 - INFO - Execute ceph nfs cluster ls -f json on 10.0.195.251
2025-04-14 16:24:14,303 - cephci - ceph:1606 - INFO - Execution of ceph nfs cluster ls -f json on 10.0.195.251 took 1.00335 seconds
2025-04-14 16:24:14,303 - cephci - nfs_ro_rw_access:81 - INFO - ceph nfs cluster created successfully
2025-04-14 16:24:14,305 - cephci - ceph:1576 - INFO - Execute ceph fs ls --format json-pretty on 10.0.195.251
2025-04-14 16:24:15,308 - cephci - ceph:1606 - INFO - Execution of ceph fs ls --format json-pretty on 10.0.195.251 took 1.003285 seconds
2025-04-14 16:24:15,310 - cephci - ceph:1576 - INFO - Execute ceph nfs export create cephfs cephfs-nfs /export_949 cephfs path=/ on 10.0.195.251
2025-04-14 16:24:16,313 - cephci - ceph:1606 - INFO - Execution of ceph nfs export create cephfs cephfs-nfs /export_949 cephfs path=/ on 10.0.195.251 took 1.003106 seconds
2025-04-14 16:24:16,315 - cephci - ceph:1576 - INFO - Execute ceph nfs export get cephfs-nfs /export_949 > export.conf on 10.0.195.251
2025-04-14 16:24:17,318 - cephci - ceph:1606 - INFO - Execution of ceph nfs export get cephfs-nfs /export_949 > export.conf on 10.0.195.251 took 1.003036 seconds
2025-04-14 16:24:17,320 - cephci - ceph:1576 - INFO - Execute cat export.conf on 10.0.195.251
2025-04-14 16:24:18,323 - cephci - ceph:1606 - INFO - Execution of cat export.conf on 10.0.195.251 took 1.003153 seconds
2025-04-14 16:24:18,324 - cephci - nfs_ro_rw_access:114 - INFO - config file for /export_949: {
  "access_type": "RW",
  "clients": [],
  "cluster_id": "cephfs-nfs",
  "export_id": 3,
  "fsal": {
    "cmount_path": "/",
    "fs_name": "cephfs",
    "name": "CEPH",
    "user_id": "nfs.cephfs-nfs.cephfs.5d839b2a"
  },
  "path": "/",
  "protocols": [
    3,
    4
  ],
  "pseudo": "/export_949",
  "security_label": true,
  "squash": "none",
  "transports": [
    "TCP"
  ]
}

2025-04-14 16:24:18,324 - cephci - nfs_ro_rw_access:115 - INFO - Make export as Readonly
2025-04-14 16:24:18,326 - cephci - ceph:1576 - INFO - Execute sed -i 's/RW/RO/g' export.conf on 10.0.195.251
2025-04-14 16:24:19,329 - cephci - ceph:1606 - INFO - Execution of sed -i 's/RW/RO/g' export.conf on 10.0.195.251 took 1.003283 seconds
2025-04-14 16:24:19,331 - cephci - ceph:1576 - INFO - Execute cat export.conf on 10.0.195.251
2025-04-14 16:24:20,335 - cephci - ceph:1606 - INFO - Execution of cat export.conf on 10.0.195.251 took 1.003491 seconds
2025-04-14 16:24:20,335 - cephci - nfs_ro_rw_access:124 - INFO - config file for /export_949: {
  "access_type": "RO",
  "clients": [],
  "cluster_id": "cephfs-nfs",
  "export_id": 3,
  "fsal": {
    "cmount_path": "/",
    "fs_name": "cephfs",
    "name": "CEPH",
    "user_id": "nfs.cephfs-nfs.cephfs.5d839b2a"
  },
  "path": "/",
  "protocols": [
    3,
    4
  ],
  "pseudo": "/export_949",
  "security_label": true,
  "squash": "none",
  "transports": [
    "TCP"
  ]
}

2025-04-14 16:24:20,335 - cephci - nfs_ro_rw_access:125 - INFO - Apply the config
2025-04-14 16:24:20,337 - cephci - ceph:1576 - INFO - Execute ceph nfs export apply cephfs-nfs -i export.conf on 10.0.195.251
2025-04-14 16:24:21,340 - cephci - ceph:1606 - INFO - Execution of ceph nfs export apply cephfs-nfs -i export.conf on 10.0.195.251 took 1.003643 seconds
2025-04-14 16:24:21,342 - cephci - ceph:1576 - INFO - Execute mkdir -p /mnt/nfs_UOPDC on 10.0.195.251
2025-04-14 16:24:22,346 - cephci - ceph:1606 - INFO - Execution of mkdir -p /mnt/nfs_UOPDC on 10.0.195.251 took 1.003306 seconds
2025-04-14 16:24:22,348 - cephci - ceph:1576 - INFO - Execute mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_949 /mnt/nfs_UOPDC on 10.0.195.251
2025-04-14 16:26:27,485 - cephci - ceph:1606 - INFO - Execution of mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_949 /mnt/nfs_UOPDC on 10.0.195.251 took 125.136587 seconds
2025-04-14 16:26:27,486 - cephci - retry:28 - WARNING - mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_949 /mnt/nfs_UOPDC returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251, Retrying in 60 seconds...
2025-04-14 16:27:27,496 - cephci - ceph:1576 - INFO - Execute mkdir -p /mnt/nfs_UOPDC on 10.0.195.251
2025-04-14 16:27:28,501 - cephci - ceph:1606 - INFO - Execution of mkdir -p /mnt/nfs_UOPDC on 10.0.195.251 took 1.003155 seconds
2025-04-14 16:27:28,502 - cephci - ceph:1576 - INFO - Execute mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_949 /mnt/nfs_UOPDC on 10.0.195.251
2025-04-14 16:29:33,634 - cephci - ceph:1606 - INFO - Execution of mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_949 /mnt/nfs_UOPDC on 10.0.195.251 took 125.13144 seconds
2025-04-14 16:29:33,635 - cephci - retry:28 - WARNING - mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_949 /mnt/nfs_UOPDC returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251, Retrying in 120 seconds...
2025-04-14 16:31:33,692 - cephci - ceph:1576 - INFO - Execute mkdir -p /mnt/nfs_UOPDC on 10.0.195.251
2025-04-14 16:31:34,697 - cephci - ceph:1606 - INFO - Execution of mkdir -p /mnt/nfs_UOPDC on 10.0.195.251 took 1.004166 seconds
2025-04-14 16:31:34,699 - cephci - ceph:1576 - INFO - Execute mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_949 /mnt/nfs_UOPDC on 10.0.195.251
2025-04-14 16:33:39,835 - cephci - ceph:1606 - INFO - Execution of mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_949 /mnt/nfs_UOPDC on 10.0.195.251 took 125.135813 seconds
2025-04-14 16:33:39,835 - cephci - nfs_ro_rw_access:168 - ERROR - mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_949 /mnt/nfs_UOPDC returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251
2025-04-14 16:33:39,836 - cephci - nfs_ro_rw_access:169 - ERROR - Traceback (most recent call last):
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_nfs/nfs_ro_rw_access.py", line 131, in run
    rc = fs_util.cephfs_nfs_mount(
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/utility/retry.py", line 32, in f_retry
    return f(*args, **kwargs)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_utilsV1.py", line 3277, in cephfs_nfs_mount
    client.exec_command(sudo=True, cmd=command)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 2305, in exec_command
    return self.node.exec_command(cmd=cmd, **kw)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 1700, in exec_command
    raise CommandFailed(
ceph.ceph.CommandFailed: mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_949 /mnt/nfs_UOPDC returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251

2025-04-14 16:33:39,836 - cephci - nfs_ro_rw_access:172 - INFO - Cleaning up the system
2025-04-14 16:33:39,838 - cephci - ceph:1576 - INFO - Execute rm -rf /mnt/nfs_UOPDC/* on 10.0.195.251
2025-04-14 16:33:40,841 - cephci - ceph:1606 - INFO - Execution of rm -rf /mnt/nfs_UOPDC/* on 10.0.195.251 took 1.003307 seconds
2025-04-14 16:33:40,843 - cephci - ceph:1576 - INFO - Execute umount /mnt/nfs_UOPDC on 10.0.195.251
2025-04-14 16:33:41,847 - cephci - ceph:1606 - INFO - Execution of umount /mnt/nfs_UOPDC on 10.0.195.251 took 1.003995 seconds
2025-04-14 16:33:41,848 - cephci - run:854 - ERROR - umount /mnt/nfs_UOPDC returned umount: /mnt/nfs_UOPDC: not mounted.
 and code 32 on 10.0.195.251
Traceback (most recent call last):
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/run.py", line 839, in run
    rc = test_mod.run(
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_nfs/nfs_ro_rw_access.py", line 179, in run
    client1.exec_command(sudo=True, cmd=command)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 2305, in exec_command
    return self.node.exec_command(cmd=cmd, **kw)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 1700, in exec_command
    raise CommandFailed(
ceph.ceph.CommandFailed: umount /mnt/nfs_UOPDC returned umount: /mnt/nfs_UOPDC: not mounted.
 and code 32 on 10.0.195.251
2025-04-14 16:33:41,851 - cephci - ceph:1576 - INFO - Execute podman --version | awk {'print $3'} on 10.0.195.21
2025-04-14 16:33:42,855 - cephci - ceph:1606 - INFO - Execution of podman --version | awk {'print $3'} on 10.0.195.21 took 1.003508 seconds
2025-04-14 16:33:42,855 - cephci - run:1065 - INFO - Podman Version 5.2.2
2025-04-14 16:33:42,856 - cephci - ceph:1576 - INFO - Execute docker --version | awk {'print $3'} on 10.0.195.21
2025-04-14 16:33:43,861 - cephci - ceph:1606 - INFO - Execution of docker --version | awk {'print $3'} on 10.0.195.21 took 1.003877 seconds
2025-04-14 16:33:43,863 - cephci - ceph:1576 - INFO - Execute ceph --version | awk '{print $3}' on 10.0.195.251
2025-04-14 16:33:44,867 - cephci - ceph:1606 - INFO - Execution of ceph --version | awk '{print $3}' on 10.0.195.251 took 1.003377 seconds
2025-04-14 16:33:44,867 - cephci - run:1081 - INFO - ceph Version 19.2.1-126.el9cp
2025-04-14 16:33:44,869 - cephci - run:1040 - INFO - ceph_clusters_file rerun/regression-vHr823-FUTHJO
2025-04-14 16:33:44,869 - cephci - run:927 - INFO - Test <module 'cephfs_nfs.nfs_ro_rw_access' from '/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_nfs/nfs_ro_rw_access.py'> failed
